vault/core.go

```diff
@@ -3052,23 +3066,34 @@ func (c *Core) readFeatureFlags(ctx context.Context) (*FeatureFlags, error) {
 // with misconfigured plugins. It should not be possible for existing builtins
 // to be misconfigured, so that is a fatal error.
 func (c *Core) isMountable(ctx context.Context, entry *MountEntry, pluginType consts.PluginType) bool {
```

Comment:
I'm not sure how much value there is in having two functions, one of which just returns the inverse of the other.

Reply:
There is value in this function being self-documenting. @tomhjp and I discussed this approach after seeing error churn related to the non-builtin-plugin lazy mounts. We thought it would be worthwhile to have this very clearly documented and central function to prevent future regressions.

Reply:
It seems to me like you're just implying that isMountable is a bad name, so why not simply rename it?

Reply:
I think I was more referring to the nature of this PR: https://github.com/hashicorp/vault/pull/18189/files and the desire to leave `isMountable`, even if it's just the inverse of another function.